autoshop/api/v1/views.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework.generics import ListAPIView, RetrieveAPIView, UpdateAPIView, DestroyAPIView
+
+from autoshop.api.v1.serializers import UserProfileSerializer
+from core.models import Profile
+
+
+class ProfileList(ListAPIView):
```

Comment:
How do we determine which approach is better than the other? They all seem to achieve the desired output.